### PR TITLE
Added Angular.js usage example.

### PR DESCRIPTION
**Pull request by hrajchert**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -145,6 +145,16 @@ if (screenfull.enabled) {
 
 See the [demo](http://sindresorhus.com/screenfull.js) for more examples, and view the source.
 
+#### Fullscreen an element with Angujar.js
+
+You can use the [Angular.js bindings](https://github.com/hrajchert/angular-screenfull) to do something like
```
Comment (sindresorhus) [inline readme.md]: Needs a `:` at the end.

File: readme.md [hunk 2]
```diff
@@ -145,6 +145,16 @@ if (screenfull.enabled) {
 
 See the [demo](http://sindresorhus.com/screenfull.js) for more examples, and view the source.
 
+#### Fullscreen an element with Angujar.js
```
Comment (sindresorhus) [inline readme.md]: typo in angular

File: readme.md (hunk 1 above)
Comment (sindresorhus) [inline readme.md]: bindings => binding ?

File: readme.md (hunk 2 above)
Comment (hrajchert) [inline readme.md]: lol... will fix

**Comment by hrajchert** [comment]
PR updated

**Comment by arthurvr** [comment]
Do we really want examples of how to use it with a specific framework? I feel like how to use easy libraries in combination with a specific framework isn't something the library should document - rather something the framework's community should be aware of. 

I feel like if we start doing this we can add examples for every JavaScript framework out there...

**Comment by sindresorhus** [comment]
I would usually agree, but Angular is super popular, so why not. If we get many more I'll just move them to links at the bottom instead.

**Pull request closed without merging**